doc/en/components/grids/grid/overview.md

```diff
@@ -107,22 +133,22 @@ Now that we have the grid packages imported, let’s get started with the basic
 <igc-grid id="grid1" auto-generate="true"></igc-grid>
 ```
 
-```ts
+```typescript
 constructor() {
-    let grid1 = (document.getElementById("grid1") as IgcGridComponent);
+    let grid1 = document.getElementById("grid1") as IgcGridComponent;
     grid1.data = data;
 }
 ```
 
 The `Id` property is a string value and is the unique identifier of the grid which will be auto-generated if not provided, while **data** binds the grid, in this case to local data.
 
-The `AutoGenerate` property tells the grid to auto generate the grid's `Column` components based on the data source fields. It will also try to deduce the appropriate data type for the column if possible. Otherwise, the developer needs to explicitly define the columns and the mapping to the data source fields.
```

Comment:
Depending on the API reference approach from above, this might need to stay or be updated as an actual ref of some sort.
Also, since there's no snippet for Blazor - this section is sitting with the text describing nothing :(
https://staging.infragistics.com/products/ignite-ui-blazor/blazor/components/grids/grid/overview#usage
Can we add an equivalent razor setup?

Reply:
Pretty sure the api refs are injected automatically for `code` words in backticks.